src/components/Tag/Tag.module.css

```diff
@@ -0,0 +1,94 @@
+@import '../../design-tokens/mixins.css';
+
+/*------------------------------------*\
+    #TAGS
+\*------------------------------------*/
+
+/**
+ * The rounded rectangle container that houses information.
+ * 1) Grabs colors from whichever variant is selected.
+ * 2) Rounds the corners of the tag.
+ */
+.tag {
+  width: max-content;
+  max-width: 20rem;
+  padding-top: var(--eds-size-half);
+  padding-bottom: var(--eds-size-half);
+  padding-left: var(--eds-size-1-and-half);
+  padding-right: var(--eds-size-1-and-half);
+
+  display: inline-flex;
+  align-items: center;
+
+  border: var(--tag-secondary-color) solid var(--eds-theme-border-width); /* 1 */
+  border-radius: 9999px; /* 2 */
+
+  background-color: var(--tag-secondary-color); /* 1 */
+  color: var(--tag-primary-color);
+  --icon-size: 1.25rem;
```

Comment:
I think we should keep this `--icon-size-default`. Icon size follows this logic:
```
var(--svg-icon-size, var(--svg-icon-size-default, 1em));
```
(in order of priority from left to right).

So this would mean:
- By default, icons in the tag will be 1.25rem
- But if someone chooses to pass in an Icon with a specific size, that will then update the value of `--icon-size`, which is highest priority and will thus be used

I should add that to CODE_GUIDELINES

Reply:
Right, saw that coming in from the Icon pr. Will change to `--icon-size-default`